src/app/template/index.ts

```diff
@@ -210,6 +210,12 @@ export class ChartTemplateBuilder {
             this.addColumn(inference.dataSource.table, key);
           });
         }
+        if (inference.disableAutoMax !== undefined) {
+          inference.disableAutoMax = false;
+        }
+        if (inference.disableAutoMin !== undefined) {
+          inference.disableAutoMin = false;
+        }
```

Comment:
Won't this force everything to `false` unless `inference.disableAutoMin` is `undefined`?  If `inference.disableAutoMin = true`, this would force it to `false`.  Is that what we want?

Reply:
Same question goes for the `inference.disableAutoMax` above.

Reply:
Thanks David. While fixing I found issue and I reworked